apps/client/src/pages/leaderboard/Leaderboard.page.tsx

```diff
@@ -0,0 +1,19 @@
+import { ReactElement, useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+export function Leaderboard(): ReactElement {
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    // after 2 seconds redirect to room
+    setTimeout(() => {
+      navigate('../room');
+    }, 2000);
+  }, []);
```

Comment:
Shall we return a callback to clear the timeout

Reply:
added cleanup function 063125405e6f093999cfaf77d5e479406ddea7a9